docs/hub/datasets-viewer-sql-console.md

```diff
@@ -20,5 +20,111 @@ Through the SQL Console, you can:
 - Embed the results of the query in your own webpage using an iframe 
 
 <Tip>
-You can also use a local DuckDB CLI to query the dataset via the `hf://` protocol. See the <a href="https://huggingface.co/docs/hub/en/datasets-duckdb" target="_blank" rel="noopener noreferrer">DuckDB CLI documentation</a> for more information.
-</Tip>
\ No newline at end of file
+You can also use the DuckDB CLI to query the dataset via the `hf://` protocol. See the <a href="https://huggingface.co/docs/hub/en/datasets-duckdb" target="_blank" rel="noopener noreferrer">DuckDB CLI documentation</a> for more information. The SQL Console provides a convenient `Copy to DuckDB CLI` button that generates the SQL query for creating views and executing your query in the DuckDB CLI.
```

Comment:
the DuckDB CLI-> a local DuckDB CLI again?

Reply:
Yes, probably could make this a little better 🤔